### PR TITLE
Update organizationMembers.js

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | The list wasn't up to date
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | 
| Sponsor company   | PrestaShop SA
| How to test?      |

On the contributors website, the "organization member icon" redirects to https://www.prestashop-project.org/project-organization/people-and-roles/  And according to this page, the list has to be updated.

### CODE REVIEW

File: constants/organizationMembers.js [hunk 1]
```diff
@@ -1,40 +1,53 @@
...
-  sarahdib: {},
-
+  sallemiines: {},
+  paulnoelcholot: {},
+  AchrafKchaou1991: {},
+  szavadskiy: {},
+  boubkerbribri: {},
   nesrineabdmouleh: {},
+  cfarhani06: {},
+  Progi1984: {},
```
Comment (Progi1984) [inline constants/organizationMembers.js:50]:
```suggestion
```

I'm only a committer. Please remove me.

File: constants/organizationMembers.js [hunk 2]
```diff
@@ -1,40 +1,32 @@
...
+  sowbiba: {},
   'matthieu-rolland': {},
-  eternoendless: {},
   PululuK: {},
   atomiix: {},
   NeOMakinG: {},
-
-  Hlavtox: {},
-
-  zuk3975: {},
```
Comment (eternoendless) [inline constants/organizationMembers.js:17]: he's still part of the project

File: constants/organizationMembers.js [hunk 3]
```diff
@@ -1,40 +1,32 @@
...
-  mparvazi: {},
```
Comment (eternoendless) [inline constants/organizationMembers.js:18]: and he

File: constants/organizationMembers.js (hunk 2 above)
Comment (nicosomb) [reply constants/organizationMembers.js:17]: OK, but @Progi1984 told me that committers shouldn't be listed here.

File: constants/organizationMembers.js [hunk 4]
```diff
@@ -1,40 +1,32 @@
...
-  marsaldev: {},
-  okom3pom: {},
-  SharakPL: {},
```
Comment (eternoendless) [inline constants/organizationMembers.js:21]: and he

**Review by eternoendless — COMMENTED** [review]
Consider sorting the list alphabetically

File: constants/organizationMembers.js (hunk 2 above)
Comment (eternoendless) [reply constants/organizationMembers.js:17]: don't they? why not?

Comment (nicosomb) [reply constants/organizationMembers.js:17]: I don't know what are the rules for this configuration. I thought it was the users on https://www.prestashop-project.org/project-organization/people-and-roles/ ...

**Comment by nicosomb** [comment]
@eternoendless tell me what the rules are and I will update the PR.

**Comment by eternoendless** [comment]
Okay, having [a look at how it's used](https://github.com/search?q=repo%3APrestaShop%2FTopContributors+organizationMembers&type=code), my conclusion is that everyone that is on the project member's list should be on this list too.

**Review by LouisAUTHIE — APPROVED** [review]

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @nicosomb

**Pull request merged**